From bf606ae577cd9142517a2170e14d4f9377a49deb Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Sat, 4 Aug 2018 07:05:17 +0100
Subject: stress-ng.h: rename statx_timestamp to shim_statx_timestamp

Newer versions of glibc now habe statx_timestamp defined, so
rename this struct so it won't clash.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 stress-ng.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/stress-ng.h b/stress-ng.h
index ae03a7b..b9359a4 100644
--- a/stress-ng.h
+++ b/stress-ng.h
@@ -2932,7 +2932,7 @@ struct shim_sched_attr {
 #define SHIM_STATX_BTIME             0x00000800U
 #define SHIM_STATX_ALL               0x00000fffU
 
-struct statx_timestamp {
+struct shim_statx_timestamp {
         int64_t		tv_sec;
         int32_t		tv_nsec;
         int32_t		__reserved;
@@ -2952,10 +2952,10 @@ struct shim_statx {
         uint64_t   stx_size;
         uint64_t   stx_blocks;
         uint64_t   __spare1[1];
-        struct statx_timestamp  stx_atime;
-        struct statx_timestamp  stx_btime;
-        struct statx_timestamp  stx_ctime;
-        struct statx_timestamp  stx_mtime;
+        struct shim_statx_timestamp  stx_atime;
+        struct shim_statx_timestamp  stx_btime;
+        struct shim_statx_timestamp  stx_ctime;
+        struct shim_statx_timestamp  stx_mtime;
         uint32_t   stx_rdev_major;
         uint32_t   stx_rdev_minor;
         uint32_t   stx_dev_major;
-- 
cgit v0.11.2

